### DISCUSSION TITLE
Random mod is now different, making an old replay broken

### DISCUSSION THREAD

**Discussion by JacksonChen666** [ppy/osu#24596] [Q&A] [answered]
[replays.zip](https://github.com/ppy/osu/files/12388269/replays.zip), [map](https://osu.ppy.sh/beatmapsets/24808#osu/84651)

The replays have the same mod (down to the attributes), but not the version. The older replay seems to miss pretty much everything. Current replay seems normal.

Current version of my lazer install is 2023.815.0-lazer. Version for the old replay is unknown.

My guess is that the random mod has now changed since, and the replay was probably made before random mod changed with an additional "Angle sharpness" setting.

**Comment by bdach** [comment, marked as answer]
Yes, we have broken random mod multiple times and each time we explicitly accepted the fact that this will break old replays. Until scoring is finalised and leaderboards are finalised this is going to be how we operate wrt mods.

**Reply by JacksonChen666** [reply]
> wrt mods

wrt?

**Reply by bdach** [reply]
https://en.wiktionary.org/wiki/WRT#English
